template/amber/repos.amber

```diff
@@ -22,8 +22,7 @@ block content
                             div.card-header
                                 img.avatar[src=$repo.Avatar]
                             div.card-block
-                                h3.login #{$repo.Name}
-                                div.full_name.hidden #{$repo.FullName}
+                                h3.full_name #{$repo.FullName}
```

Comment:
does changing the css class to `full_name` alter the style? Not sure if we have a class for `full_name` in which case we should leave the class as `login`